src/Controls/tests/DeviceTests/HandlerTestBase.Android.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Android.Views;
+
+namespace Microsoft.Maui.DeviceTests
+{
+	public partial class HandlerTestBase
+	{
+		protected bool GetIsAccessibilityElement(IViewHandler viewHandler) =>
+			GetSemanticNativeElement(viewHandler).ImportantForAccessibility == ImportantForAccessibility.Yes;
+
+
+		protected bool GetExcludedWithChildren(IViewHandler viewHandler) =>
+			GetSemanticNativeElement(viewHandler).ImportantForAccessibility == ImportantForAccessibility.NoHideDescendants;
+
+		public View GetSemanticNativeElement(IViewHandler viewHandler)
+		{
+			if (viewHandler.NativeView is AndroidX.AppCompat.Widget.SearchView sv)
```

Comment:
Why is SearchView a special case again?

Reply:
In android that's the view that gets read out when navigating to the searchview.

If you look at the `MappingSemantics` call we extract out the button and that's what we apply the properties to

https://github.com/dotnet/maui/blob/9396975987e213e7e9a7872898cae6d5acfbb6ff/src/Core/src/Handlers/View/ViewHandler.Android.cs#L91